### DISCUSSION TITLE
Help to calibrate gain and offset

### DISCUSSION THREAD

**Discussion by floxia** [yasir-shahzad/CS5530#5] [Q&A]
Hi there,

Thank you for the incredible work you have done creating this library. 

I have been using this library with great success and I am now trying to dig deeper and I have noticed that the chip offers gain and offest calibration. Could you show an example of how to achieve that using your library? What kind of addresses do I need to put into the calibrate function?

Thank you very much!

**Comment by yasir-shahzad** [comment]
Sorry for late reply, i will put examples for that in 2days
